content/pt/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,387 @@
+kubectl config set-credentials kubeuser/foo.kubernetes.com --username=kubeuser --password=kubepassword
+
+# salve o namespace permanentemente para todos os comandos subsequentes do kubectl nesse contexto.
+kubectl config set-context --current --namespace=ggckad-s2
+
+# defina um contexto utilizando um nome de usuário e o namespace.
+kubectl config set-context gce --user=cluster-admin --namespace=foo \
+  && kubectl config use-context gce
+ 
+kubectl config unset users.foo                       # excluir usuário foo
+```
+
+## Aplicar
+`apply` gerencia aplicativos através de arquivos que definem os recursos do Kubernetes. Ele cria e atualiza recursos em um cluster através da execução `kubectl apply`.
+Esta é a maneira recomendada de gerenciar aplicativos Kubernetes em ambiente de produção. Veja a [documentação do Kubectl](https://kubectl.docs.kubernetes.io).
+
+## Criando objetos
+
+Manifestos Kubernetes podem ser definidos em YAML ou JSON. A extensão do arquivo `.yaml`,
+`.yml`, e `.json` pode ser usado.
+
+```bash
+kubectl apply -f ./my-manifest.yaml            # criar recurso (s)
+kubectl apply -f ./my1.yaml -f ./my2.yaml      # criar a partir de vários arquivos
+kubectl apply -f ./dir                         # criar recurso (s) em todos os arquivos de manifesto no diretório
+kubectl apply -f https://git.io/vPieo          # criar recurso (s) a partir de URL
+kubectl create deployment nginx --image=nginx  # iniciar uma única instância do nginx
+kubectl explain pods,svc                       # obtenha a documentação para pod e svc
+
+# Crie vários objetos YAML a partir de stdin
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-sleep
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    args:
+    - sleep
+    - "1000000"
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-sleep-less
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    args:
+    - sleep
+    - "1000"
+EOF
+
+# Crie um segredo com várias chaves
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  password: $(echo -n "s33msi4" | base64 -w0)
+  username: $(echo -n "jane" | base64 -w0)
+EOF
+
+```
+
+## Visualizando e localizando recursos
+
+```bash
+# Obter comandos com saída simples
+kubectl get services                          # Listar todos os serviços do namespace
+kubectl get pods --all-namespaces             # Listar todos os pods em todos namespaces
+kubectl get pods -o wide                      # Listar todos os pods no atual namespace, com mais detalhes
+kubectl get deployment my-dep                 # Listar uma implantação específica
+kubectl get pods                              # Listar todos os pods no namespace
+kubectl get pod my-pod -o yaml                # Obter o YAML de um pod
+kubectl get pod my-pod -o yaml --export       # Obtenha o YAML de um pod sem informações específicas do cluster
```

Comment:
Can you please review this line. Its not defined in the online doc.

Reply:
Line removed